include/boost/geometry/algorithms/detail/equals/collect_vectors.hpp

```diff
@@ -327,17 +326,13 @@ struct range_collect_vectors
 
     static inline void apply(Collection& collection, Range const& range)
     {
-        typedef geometry::detail::normalized_view
-            <
-                Range const
-            > normalized_range_type;
-
-        apply_impl(collection, normalized_range_type(range));
+        apply_impl(collection,
+                   detail::closed_clockwise_view<Range const>(range));
     }
 
 private:
-    template <typename NormalizedRange>
-    static inline void apply_impl(Collection& collection, NormalizedRange const& range)
+    template <typename RevClRange>
```

Comment:
What `RevClRange` means? Reverse closed or reversed clockwise range? `normalized_view` was renamed to  `closed_clockwise_view` should NormalizedRange be renamed accordingly, or I misunderstood something?

Reply:
Yes, I'll change the name.